src/Bicep.Core/Semantics/IBinder.cs

```diff
@@ -13,6 +15,11 @@ public interface IBinder : ISyntaxHierarchy
 
         FileSymbol FileSymbol { get; }
 
+        Uri FileUri { get; }
+
+        IFileResolver FileResolver { get; }
```

Comment:
>FileResolver [](http://example.com/codeflow?start=22&length=12)

nit - extra line?

Reply:
I removed FileReolver from Binder, and moved to SemanticModel.